src/main/java/com/smartsheet/api/internal/json/JacksonJsonSerializer.java

```diff
@@ -332,4 +365,31 @@ public JacksonJsonSerializer() {
 		}
 		return result;
 	}
+
+	/**
+	 * De-serialize to a CopyOrMoveRowResult object from JSON
+	 * @param inputStream
+	 * @return
+	 * @throws JSONSerializerException
+	 */
+	@Override
+	public CopyOrMoveRowResult deserializeCopyOrMoveRow(java.io.InputStream inputStream) throws JSONSerializerException{
```

Comment:
Curious why there is a specific method for CopyOrMoveRow, where everyone else goes through the standard deserialization methods